Tests/ProcedureKitTests/KVONotificationTests.swift

```diff
@@ -0,0 +1,378 @@
+//
+//  ProcedureKit
+//
+//  Copyright © 2016 ProcedureKit. All rights reserved.
+//
+
+import XCTest
+import TestingProcedureKit
+@testable import ProcedureKit
+
+class KVOTests: ProcedureKitTestCase {
```

Comment:
It's great to see this huge source of uncertainty finally locked down with some tests.